projects/next-range-selector/src/lib/next-range-selector.component.ts

```diff
@@ -187,13 +171,13 @@ export class NextRangeSelectorComponent implements OnInit, ControlValueAccessor
   @Input() public process?: ProcessProp = true;
   @Input() public lazy = false;
   @Input() public duration: number = 0.5;
+  @Input() public tabIndex: number = 1;
```

Comment:
Are you sure that tabIndex work correct? I've tried change it for first slider in app.html (last version with 8 merged PR) on [tabIndex]="2". It works not like I expected.